### DISCUSSION TITLE
filamentphp implementation

### DISCUSSION THREAD

**Discussion by zxhub-llc** [tenancy/tenancy#301] [Q&A]
Hello, how are you, dear developer, I have a question about the implementation of the service in filament since in the filament documentation they manage their own implementation of multi tenancy, but they do not manage all the functionalities that you have, and I would like to know if they have some documentation on the implementation
<img width="1680" alt="Screen Shot 2024-05-14 at 01 03 27" src="https://github.com/tenancy/tenancy/assets/161612251/cff20874-829c-4103-bd85-53b871fde17c">
![Uploading Screen Shot 2024-05-14 at 01.03.31.png…]()

**Comment by ArlonAntonius** [comment]
Hey @zxstudio-llc,

There is no direct documentation that I can provide you with to integrate Tenancy with filament at this moment. I do really like the idea of it and will be creating one when I have the time.
In general it should simply be installing Tenancy as well and following the installation guide for both.

If there are any questions you have, feel free to reach out!
